runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotConfig.java

```diff
@@ -132,11 +132,22 @@ default boolean screenshotKills()
 		return false;
 	}
 
+	@ConfigItem(
+			keyName = "usernameFileName",
```

Comment:
Also, I think this toggle is not necessary, so instead of fixing the indentation, simply delete it

Reply:
> Also, I think this toggle is not necessary, so instead of fixing the indentation, simply delete it

I made these two corrections and Nightfirecat was able to do some testing and discovered that there are no scenarios where a player receives a kill message without a onPlayerLootReceived event being triggered so let me know if everything looks good